### PR TITLE
Add CMakeCache syntax highlighting

### PR DESCRIPTION
**Pull request by rwols**
I have this syntax in my CMakeBuilder plugin, but I think it's better if this plugin has it. It provides highlighting for CMakeCache.txt files.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,9 +1,12 @@
-# Sublime Text 2/3 - CMake Package #
+# CMake
```
Comment (zyxar) [inline README.md]: I don't think it necessary to change the title.

Comment (rwols) [inline README.md]: OK, I'll revert that

**Comment by rwols** [comment]
I have more stuff to add to this PR (more custom command handling), but I've been busy with other things, please don't merge this yet.

File: CMakeCache.sublime-syntax [hunk 2]
```diff
@@ -0,0 +1,54 @@
...
+        - meta_scope: comment.line.number-sign.cmakecache
+        - match: $
+          pop: true
+
+  key-value-pair:
+    - match: ^(\w|-)+
+      scope: variable.parameter.cmakecache
+      set: expect-type
+    - match: .
+      scope: invalid.illegal
```
Comment (rwols) [inline CMakeCache.sublime-syntax]: I forgot to put a .cmakecache prefix here

**Pull request merged**